### ISSUE TITLE
[fvt]2.12.1 xcatprobe image -d need modified if no compute set diskless image 

### ISSUE THREAD

**Issue by junxiawang** [xcat2/xcat-core#1402]
env:ubuntu
build:root@c910f03c05k05:/opt/xcat/bin# lsxcatd -v
Version 2.12.1 (git commit 5431a2ab9ccd42f5f76540ec0868429dd6199056, built Sun Jun 26 06:47:12 EDT 2016)
root@c910f03c05k05:/opt/xcat/bin# 
How to reproduce:
```
root@c910f03c05k05:/opt/xcat/bin# xcatprobe image -d
Smartmatch is experimental at /opt/xcat/bin/xcatprobe line 64.
Smartmatch is experimental at /opt/xcat/bin/xcatprobe line 173.
root@c910f03c05k05:/opt/xcat/bin# echo $?
0
root@c910f03c05k05:/opt/xcat/bin# lsxcatd -v

```
-----------------> need to return  Invalid parameter for image

**Comment by junxiawang** [comment]
only diskfull image on the node ,so u should give information  but no output here.

**Comment by gurevichmark** [comment]
I am already making changes to `image` probe based on Xiaopeng's suggestions. Will fix this under the same pull request.

**Comment by gurevichmark** [comment]
Fixed under #1406
